src/Bicep.Cli.IntegrationTests/Bicep.Cli.IntegrationTests.csproj

```diff
@@ -22,6 +22,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Update="Microsoft.CodeAnalysis.BannedApiAnalyzers" Version="3.3.4" />
```

Comment:
Does this package version need to be updated in [src/Directory.Build.props](https://github.com/Azure/bicep/blob/051bf85361f57a988aa536d1538248465b2cefe6/src/Directory.Build.props#L51) along with (instead of?) the updates to all csproj files?

Reply:
Yeah it does. There shouldn't be added lines for that package. I'll push an update.

Reply:
`Nerdbank.GitVersioning` has the same issue, so I ended up removing it as well. (Is present in Directory.Build.props and was using `Update` instead of `Include` in project files.)